lib/rubocop/cop/mixin/enforce_superclass.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 module RuboCop
-  module Cop
+  module Cop # rubocop:disable Style/Documentation
+    remove_const(:EnforceSuperclass) if defined?(EnforceSuperclass)
```

Comment:
There are three warnings printed about methods of `EnforceSuperclass` being redefined. Rather than undefining each of those three methods individually, here I just undefine the whole `EnforceSuperclass` module (if it is already defined). We will then (re-)define it just below.